Automattic/wp-calypso
# Add link in bio `edit links` completion logic

#### Proposed Changes

These changes only affect the `link-in-bio` launchpad flow.

* **Disable the `Launch Link bio` until the user completes the `Add links` task**

Before completing `Add links` task:

<img width="458" alt="image" src="https://user-images.githubusercontent.com/10482592/187726296-90dde9cb-36c6-45d2-9008-7b7906fa8c88.png">


After completing `Add links` task:

<img width="446" alt="image" src="https://user-images.githubusercontent.com/10482592/187726362-ebfd12aa-1bc3-46dc-ae1a-ae5314628c2e.png">


#### Testing Instructions



* Check out this branch and execute `yarn start` command
* Create a new `link-in-bio` site. Visit https://wordpress.com/hp-2022-tailored-flows/ and scroll down to the` Put a dent in the web.` In 3 minutes flat. subheading. You should end up on the launchpad at the end of the flow.
* Switch to local calypso and load the launchpad page again (http://calypso.localhost:3000/setup/launchpad?flow=link-in-bio&siteSlug=YOUR_SITE_SLUG)
* **The `Add links` task should be enabled and the `Launch Link in bio` task should be disabled**
* Click the `Add links` task
* Navigate back to the launchpad using the navigation sidebar in the site editor
*  **The `Add links` task should be disabled and the `Launch Link in bio` task should be enabled**

#### Pre-merge Checklist






Related to #
